S3/Utils.py

```diff
@@ -33,8 +35,8 @@
     sys.stderr.flush()
     sys.exit(EX_OSFILE)
 
-import Config
-import Exceptions
+import S3.Config
```

Comment:
In the long run, it might be better to remove all those circular import dependencies. But doing so might change some of the APIs.